frontend/src/components/ContactModel.vue

```diff
@@ -9,7 +9,7 @@
                 <!-- Modal header -->
                 <div class="flex items-start justify-between p-5 pb-0">
                     <h3 class="text-2xl font-semibold text-black lg:text-4xl">
```

Comment:
Add the `test-contact-getInTouch` class, please

Reply:
Done!